src/clr-angular/data/datagrid/render/header-renderer.spec.ts

```diff
@@ -197,8 +196,8 @@ export default function(): void {
       column3InitialWidth = widthOf(columnHeader3Element);
       column4InitialWidth = widthOf(columnHeader4Element);
 
-      columnHeader1DraggableDebugElement = context.fixture.debugElement.queryAll(By.directive(ClrDraggable))[0];
-      columnHeader3DraggableDebugElement = context.fixture.debugElement.queryAll(By.directive(ClrDraggable))[2];
+      columnHeader1DraggableDebugElement = context.fixture.debugElement.queryAll(By.directive(ClrDraggable))[1];
+      columnHeader3DraggableDebugElement = context.fixture.debugElement.queryAll(By.directive(ClrDraggable))[5];
```

Comment:
`columnHeader5DraggableDebugElement`?

Reply:
This tells me need to improve these variable names. :)